target/linux/ramips/dts/mt7621_netgear_r6700-v2.dts

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "mt7621_netgear_sercomm_chj.dtsi"
+
+/ {
+	compatible = "netgear,r6700-v2", "mediatek,mt7621-soc";
+	model = "Netgear R6700v2";
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "r6700v2:green:power";
+			gpios = <&gpio0 22 GPIO_ACTIVE_LOW>;
+		};
+
+		
+		wlan {
+			label = "r6700v2:white:wlan";
+			gpios = <&gpio0 5 GPIO_ACTIVE_HIGH>;
+		};
+
+		wps {
+			label = "r6700v2:white:wps";
+			gpios = <&gpio0 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 18 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		wifi {
+			label = "wifi";
+			gpios = <&gpio0 14 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_RFKILL>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_RESTART>;
+		};

```

Comment:
You have different keys than defined in mt7621_netgear_chj.dtsi.
I'm not sure whether it would make sense here to go back to a separate DTS file or whether one could reasonable use a common DTSI.
@blocktrron Any ideas?